fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -117,6 +122,42 @@ fun ProductAnalysis(
     }
 }
 
+@Composable
+private fun ReanalyzeCard() {
+    ReviewQualityCheckInfoCard(
+        title = stringResource(R.string.review_quality_check_reanalysis_title),
+        type = ReviewQualityCheckInfoType.Analysis,
+        modifier = Modifier.fillMaxWidth(),
+        buttonText = stringResource(R.string.review_quality_check_reanalysis_button),
+        onButtonClick = {},
+        icon = {
+            Icon(
+                painter = painterResource(id = R.drawable.mozac_ic_information_fill_24),
+                contentDescription = null,
+                modifier = Modifier.size(24.dp),
+                tint = FirefoxTheme.colors.iconPrimary,
+            )
+        },
+    )
+}
+
+@Composable
+private fun CheckCard() {
```

Comment:
what exactly is "check card"? I noticed that it's not being used outside of a preview, so not quite sure what this card is for.

Reply:
to save you some work, does this overlap with this ticket?

Reply:
This card would be displayed while reanalysis is happening, figured I'd add it to this patch since the task is to create cards related to reanalysis

Reply:
Ugh, I forgot to paste the link so my comment makes no sense 🤦‍♀️ . Sorry about that

https://bugzilla.mozilla.org/show_bug.cgi?id=1847569

Reply:
Ah, ya didn't realize that was a separate task, would you rather I remove the CheckCard from this patch or?

Reply:
I'd say remove it, if only to keep the diff cleaner. I'm gonna do my next pass by end of day though and we should be good to land this soon 👍

Reply:
Sure, removed CheckCard and its preview